core/src/main/java/org/geysermc/geyser/translator/level/block/entity/PistonBlockEntity.java

```diff
@@ -629,17 +652,25 @@ private void placeFinalBlocks() {
             return;
         }
         placedFinalBlocks = true;
+
+        BoundingBox playerBoundingBox;
```

Comment:
I see this bounding box get is done twice - might be helpful to have a method in CollisionManager for getting the active bounding box?